custom_components/ramses_cc/climate.py

```diff
@@ -222,11 +214,6 @@ def hvac_mode(self) -> str | None:
             return HVACMode.AUTO  # users can't adjust setpoints in away mode
         return HVACMode.HEAT
 
-    @property
-    def name(self) -> str:
-        """Return the name of the Controller."""
-        return "Controller"
```

Comment:
This now inherits from the HA device.

Reply:
So, primary entities:
 - every physical device (except FANs/switches?)
 - the TCS (which is also a physical device)
 - each zone
 - HW
 - FANs, and switches
 
 Is that right?

Reply:
My wording could perhaps have been a bit clearer here. What I was meaning to highlight is that this change causes these entities to just use the name of the device. Taking zones as an example there is a HA device for each zone called say "Living room" and then the climate entity for that zone is also just called "Living room" rather than "Living room Climate".

Reply:
For this system climate entity the name will be whatever we decide to name the device [here](https://github.com/zxdavb/ramses_cc/pull/121#discussion_r1451712455).